src/include/sof/lib/pm_runtime.h

```diff
@@ -48,6 +48,7 @@ enum pm_runtime_context {
 	DMIC_POW,			/**< DMIC Power */
 	DW_DMAC_CLK,			/**< DW DMAC Clock */
 	CORE_MEMORY_POW,		/**< Core Memory power */
+	DSP_D0				/**< DSP in D0 state */
```

Comment:
We should really prefix these enums, can do as a seperate patch/PR though.

Reply:
Renamed to PM_RUNTIME_DSP.